test/integration/conftest.py

```diff
@@ -50,16 +52,90 @@ def run_long_tests():
     return os.environ.get(RUN_LONG_TESTS, None)
 
 
+@pytest.fixture(autouse=True, scope="session")
+def cloud_linode_firewall(test_linode_client):
```

Comment:
Can we rename it to something like `e2e_test_firewall` or `integration_test_firewall` to clarify what it's used for?